recipes/arrow/all/conanfile.py

```diff
@@ -401,6 +404,9 @@ def generate(self):
         deps.set_property("mimalloc", "cmake_target_name", "mimalloc::mimalloc")
         deps.generate()
 
+        venv = VirtualBuildEnv(self)
+        venv.generate()
+
```

Comment:
In conan 2, this is automatic and is no longer needed :)

Reply:
Follow up for @AbrilRBS comment:

```suggestion
```